modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -55,10 +77,53 @@ function inope_policy_review_tab_title_callback($node) {
  *   policy nodes.
  */
 function inope_policy_review_page($node) {
-  // Create a build array to be displayed on the page.
-  $build = array(
-    'inope_policy_review_form' => drupal_get_form('inope_policy_review_form', $node),
-  );
+  global $user;
+  $account = $user;
+  $build = array();
+  $policy_review_id = NULL;
+
+  // Get the parameters passed in the url and if possible set policy_review_id.
+  $get_params = drupal_get_query_parameters();
+  if (!empty($get_params['id'])) {
+    $policy_review_id = $get_params['id'];
```

Comment:
I probably should have expanded my inline comments for this one but:

On line 86 I used drupal_get_query_parameters() and stored it as a variable for reuse later.  Then on line 88 the id key of the array returned from drupal_get_query_parameters is set to a variable named $policy_review_id.

Is there a way for me to tell which line was commented on?  It may help me respond to comments better in the future.